### DISCUSSION TITLE
Why is my github page "Not secure" when using custom domain?

### DISCUSSION THREAD

**Discussion by ngraciella** [community/community#21125] [Other Feature Feedback, Questions, & Ideas] [answered]
Title

But directly accessing the page using https:// works. How do I redirect http:// to https:// ?

**Comment by natashapetrus** [comment, marked as answer]
Navigate to Settings > Pages and ensure the "Enforce HTTPS" box is checked.

More info here: https://docs.github.com/en/pages/getting-started-with-github-pages/securing-your-github-pages-site-with-https

**Comment by Tormentous** [comment]
you gotta input the whole thing with https for example https://yourdomain.com/ into the custom domain field
